onboarding_documentation/technical_documentation/analyst_tutorial/analyst_tutorial.md

```diff
@@ -0,0 +1,122 @@
+
+#### Task: Write a config file
+- If we are wanting to run the `large cohort` pipeline on the `bioheart-test` dataset, we will need to create a config file that is capable of doing this.
+- Have a go at writing your own config file capable of running the `large cohort` pipeline on `bioheart-test` up until the `Combiner` stage. 
+- You can use the default config file as a starting point, and then override the necessary parameters to run the pipeline on `bioheart-test`.
+- *Hint*: You will **not** need to change anything in the default `large cohort` config file.
+
+<details>
+<summary>Click to reveal the config file</summary>
+
+```TOML
+[workflow]
+input_datasets = ['bioheart-test']
+sequencing_type = 'genome'
+output_version = '1.0' # Do we want them to specify output_version?
+only_sgs = [<list of sequencingGroup IDs>] # to be used to demonstrate how to run on a subset of samples
+```
+
+</details>
+
+Once the config file has been written, save it with an appropriate name (e.g. `bioheart-test.toml`) in a directory that makes sense on your local machine. In the following step we will be using `analysis-runner` to submit the job and point it to the config file we have just written.
+
+## 3. Submitting an analysis job
+Now that we have a config file, we can submit an analysis job. To do this we will use the `analysis-runner` tool. The `analysis-runner` tool is a command line tool that is used to submit analysis jobs to the cloud.
+
+Because we are running the `large cohort` pipeline we will need to be inside the `Production-pipelines` repository. So in a directory you want to keep `Production-pipelines` in, run the following command:
+
+```bash
+git clone --recurse-submodules git@github.com:populationgenomics/production-pipelines.git
+cd production-pipelines
+```
+
+Ensure you are on the `main` branch.
+
+For guidance on how to use the `analysis-runner` tool, please see the `analysis-runner` documentation [here](https://github.com/populationgenomics/analysis-runner). It will need to be `pip` installed before use.
+
+It is generally good practice to create a bash script that contains the `analysis-runner` command to submit the job. This is so that the command can be easily re-run if necessary.
+
+#### Task: Submit an analysis job
+- Using the instructions in the `analysis-runner` repo linked above as well as the config file you have just written, write the command to submit analysis job to the cloud using the `analysis-runner` tool.
```

Comment:
Again, there is too little explanation and background knowledge for the individual to actually get this.

Rather than tasks, I think you can just walk the individual through what needs to get done